tests/check.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/bash
+set -euo pipefail
+dn=$(dirname $0)
+srcdir=$(cd ${dn}/.. && pwd)/src
+
+# Verify syntax for sources
+for f in $(echo ${srcdir}/*); do
+    if ! [ -f "$f" ]; then continue; fi
+    shebang=$(head -1 $f)
+    if [[ $shebang =~ ^#!/usr/bin/python.* ]]; then
```

Comment:
Maybe let's just grep for `^#!/.*/python.*`? It'd be pretty easy to miss a patch that introduces a new file with `/bin/python` instead of `/usr/bin/python`.